src/components/Search.js

```diff
@@ -18,12 +18,12 @@ const Search = ({ updateStock }) => {
   const onSubmit = ({ key, target }) => {
     if (key === 'Enter') {
       toggleIsOpen(false);
+      dispatch(resetAction())
```

Comment:
❌ Major: This shouldn't be here. You're only dispatching `resetAction` when hitting Enter, and not when clicking on the options in the dropdown menu.
This should be in Redux middleware, when it is handling whatever action `updateStock` dispatches / when it is sending a server request off to get the new stock information.

Reply:
yes, I 've now changed it to middleweare